main.tf

```diff
@@ -0,0 +1,127 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+      version = ">= 3.8"
+    }
+  }
+  backend "azurerm" {
+        resource_group_name  = "Cohort28_AleTan_ProjectExercise"
+        storage_account_name = "alexstorageex13"
+        container_name       = "bacpac1"
+        key                  = "terraform.tfstate"
+    }
+}
+
+provider "azurerm" {
+  features {}
+}
+
+data "azurerm_resource_group" "main" {
+  name = var.resource_group_name
+}
+
+resource "azurerm_service_plan" "main" {
+  name                = "${var.prefix}-asp"
+  location            = data.azurerm_resource_group.main.location
+  resource_group_name = data.azurerm_resource_group.main.name
+  os_type             = "Linux"
+  sku_name            = "B1"
+}
+
+resource "azurerm_linux_web_app" "main" {
+  name                = "${var.prefix}-terraformToDo"
+  location            = data.azurerm_resource_group.main.location
+  resource_group_name = data.azurerm_resource_group.main.name
+  service_plan_id     = azurerm_service_plan.main.id
+
+  site_config {
+    application_stack {
+      docker_image     = "appsvcsample/python-helloworld"
+      docker_image_tag = "latest"
```

Comment:
This should actually match your app! And you may also see that this syntax is deprecated, so TF would prefer:
```terraform
    docker_image_name     = "appsvcsample/python-helloworld:latest"
    docker_registry_url   = "https://index.docker.io"
```
(But with your image name!)

Reply:
Done